commcare_connect/templates/opportunity/invoice_list.html

```diff
@@ -17,12 +17,12 @@
       <ul class="p-2 w-full max-h-80 overflow-y-auto">
         <li class="p-2 hover:bg-gray-100 rounded cursor-pointer text-left">
           <a href="{{ new_invoice_url }}?invoice_type=service_delivery">
-            <p class="text-brand-deep-purple text-sm font-medium">{% translate Service Delivery %}</p>
+            <p class="text-brand-deep-purple text-sm font-medium">{% translate "Service Delivery" %}</p>
```

Comment:
Ugh, meant to amend this commit to the previous one - alas...sorry for the noise.